.travis.yml

```diff
@@ -19,7 +22,7 @@ cache:
     - $HOME/.composer/cache/files
 
 before_script:
-    - composer install --no-interaction --prefer-dist
+    - composer install
```

Comment:
why removing this? I would actually add `--no-progress --no-suggest`

Reply:
Do you mean replace ```--no-progress --no-suggest``` with  ````--no-interaction --prefer-dist```` ?

Reply:
No I mean adding it as well, sorry for not being clear enough

Reply:
It's okay. I will add this options back.